packages/manager/apps/catalog-revamp/src/components/ManagerTile/ManagerTile.tsx

```diff
@@ -1,20 +1,14 @@
 import React from 'react';
```

Comment:
this component is supposed to be deleted

Reply:
Done